website/docs/docs/collaborate/data-health-signals.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Data health signals"
+sidebar_label: "Data health signals"
+id: data-health-signals
+description: "Learn how data health signals offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer."
+image: /img/docs/collaborate/dbt-explorer/data-health-signal.jpg
+---
+
+# Data health signals <Lifecycle status="preview" />
+Data health signals offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer. They keep you informed on the status of your resource's health using the indicators **Healthy**, **Caution**, **Degraded**, or **Unknown**.
+
+- Supported resources are [models](/docs/build/models), [sources](/docs/build/sources), and [exposures](/docs/build/exposures).
+- For accurate health data, ensure the resource is up-to-date and had a recent job run.
+- Each data health signal reflects key data health components, such as test success status, missing resource descriptions, missing tests, absence of builds in 30-day windows, [and more](#data-health-signal-criteria)
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-health-signal.jpg" width="55%" title="View data health signals for your models."/> 
+
+## Access data health signals
+
+Access data health signals in the following places:
+- In the [search function](/docs/collaborate/explore-projects#search-resources) or under **Models**, **Sources**, or **Exposures** in the **Resource** tab.  
+  - For sources, the data health signal also indicates the source freshness status.
```

Comment:
Hiya @mirnawong1 

This is a non-blocker suggestion / Would it be good to link out source freshness to a source freshness page: https://docs.getdbt.com/docs/deploy/source-freshness or https://docs.getdbt.com/docs/build/sources#snapshotting-source-data-freshness

Kind Regards
Natalie